inc/class-ses.php

```diff
@@ -158,6 +161,7 @@ public function send_wp_mail( $to, $subject, $message, $headers = [], $attachmen
 					],
 					'Body'   => [],
 				],
+				'ConfigurationSetName' => $this->config_set,
 			];
```

Comment:
What values are acceptable here? If the API rejects a `null` value or empty string we can set this in an additive way e.g:

```suggestion
			];

			if ( ! empty( $this->config_set ) ) {
				$args['ConfigurationSetName'] = $this->config_set;
			}
```